pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/blocks/TransferableBlock.java

```diff
@@ -41,28 +42,43 @@ public class TransferableBlock implements Block {
 
   private final BaseDataBlock.Type _type;
   private final DataSchema _dataSchema;
+  private final boolean _isErrorBlock;
```

Comment:
We don't need to explicitly track this. A block is an error block if the type is METADATA and has 1 or more exceptions in the map in the block ?

Reply:
this is only needed for transferable block --> b/c the data block is lazy generated. accessing the exception map is costly